git/mergebase.go

```diff
@@ -9,6 +9,23 @@ type MergeBaseOptions struct {
 }
 
 func MergeBase(c *Client, options MergeBaseOptions, commits []Commitish) (CommitID, error) {
+	if len(commits) == 2 {
```

Comment:
This looks good to me. Did you want to include a test case for this since it looks like it could be reproduced with a simple repo/branch with two commits in it?